api/grpc/mpi/v1/command.proto

```diff
@@ -46,7 +46,7 @@ message CreateConnectionRequest {
 // A representation of instances and runtime resource information
 message Resource {
     // A resource identifier
-    string id = 1;
+    string resource_id = 1;
```

Comment:
Is the `resource_` prefix not a little redundant?

Reply:
Ditto for all `<ENTITY_NAME>_` prefixes in this PR.

Reply:
I think we have being following this convention for other fields as well so we should probably stay consistent